docs/agents/sematext-agent/kubernetes/installation.md

```diff
@@ -107,11 +171,15 @@ spec:
         - name: LOGGING_LEVEL
           value: "info"
         - name: KUBERNETES_CLUSTER_ID
-          value: <REPLACE_WITH_CLUSTER_NAME>
+          value: "default"
```

Comment:
Will this work if they use name other name/ID for their cluster?  If not, it may be hard for non-k8s experts to figure out why things are not working.

Reply:
If I remember correctly this is just some name users would give to the cluster, not actually a setting. When at eHealth I searched for cluster name for half day before someone told me to write anything I want there. Maybe just don't mention it, put it under advanced?